package.json

```
@@ -64,6 +64,7 @@
     "vue-multiselect": "2.1.6",
     "vue-template-compiler": "2.6.12",
     "webpack": "4.46.0",
+    "webpack5": "npm:webpack@^5.42.1",
```

We generally pin our dependencies to specific versions to try to keep things reproducible.
```suggestion
    "webpack5": "npm:webpack@5.42.1",
```